repo/open.go

```diff
@@ -367,6 +370,11 @@ func openWithConfig(ctx context.Context, st blob.Storage, cliOpts ClientOptions,
 		},
 	}
 
+	dr.registerEarlyCloseFunc(func(ctx context.Context) error {
+		dr.CloseDebug(ctx)
+		return nil
+	})
```

Comment:
same suggestion as in the other place where CloseDebug() is used. Since, we are calling `debug.StartProfileBuffers` as this layer, it should be ok to call `debug.StopProfileBuffers` fro this layer as well. Thoughts ?